charts/kafka-lag-exporter/templates/050-Service.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     helm.sh/chart: {{ include "kafka-lag-exporter.chart" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+    jobLabel: kafka-lag-exporter
```

Comment:
Use one of the derived names here.  `{{ include "kafka-lag-exporter.fullname" . }}` may be sufficient.

Reply:
fixed to use derived name in templating